pkg/loki/modules_test.go

```diff
@@ -35,3 +38,30 @@ func TestUniqueDeps(t *testing.T) {
 	expected := []moduleName{Server, Overrides, Distributor, Ingester}
 	assert.Equal(t, expected, uniqueDeps(input))
 }
+
+func TestActiveIndexType(t *testing.T) {
+	var cfg chunk.SchemaConfig
```

Comment:
I think you can transform those into table tests with name.

Reply:
Sorry, I am not sure I get it. Can you please elaborate more so that we don't have to come back again on this?

Reply:
https://github.com/golang/go/wiki/TableDrivenTests

Reply:
Ohh I didn't know the term. Thanks for the link!